languages/en.ini

```diff
@@ -390,7 +390,7 @@ Excerpt = "Excerpt"
 exclude_facet = "Exclude matching results"
 exclude_newspapers = "Exclude newspaper articles"
 Export = "Export"
-Export Favorites = "Export Favorites"
+Export Favorites = "Export Seleted Items"
```

Comment:
I think this should just be "Export Items," because this is used as the title of the dialog box where the user makes the selection. Also note that, if you really feel we want the word "Selected," there is a typo here (missing "c"). :-)

Reply:
This string is only used as the [title for the cart](https://github.com/vufind-org/vufind/blob/1716403969bc00d0ad4bdaf9aa3602e790567ec8/themes/bootstrap3/templates/cart/export.phtml#L3) as far as I can tell, so I think Export Items would be more appropriate. Maybe we could change the key to Export Cart, actually. Unless I'm mistaken.

Reply:
Since this PR is just changing values and not keys, I'm going to merge it as-is since we all seem to agree on the "Export Items" wording. If we want to change keys, we can deal with that in the future; I prefer not to without really strong reason, since that's more likely to break people's local customizations.